layers/+lang/javascript/packages.el

```diff
@@ -160,6 +160,7 @@
     :defer t
     :init
     (progn
+      (spacemacs/declare-prefix-for-mode 'js2-mode "mT" "repl")
```

Comment:
The prefix here should be "toggle", repl bindings are under "ms" by convention

Reply:
@yuhan0 
Thank you!